### PR TITLE
allow any access

### PR DESCRIPTION
**Pull request by janschumann**
We have the requirement that we do not want to create a IAM role for the ES instance. Instead we only use security groups to maintain access.

With the current code this is not possible. I tried to use `iam_role_arns = ["*"]` but as the default aim role will be included, which results in ["<default_iam_role_arn", "*"], which causes an error.

I would like to introduce `create_default_iam_role`. When set to `false`, the `aws_iam_role.elasticsearch_user` will not be created. 

Additionally, when `var.iam_role_arns` is empty, the access policy will allow any AWS identifiers (`["*"]`) to access the domain.

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -235,3 +241,11 @@ variable "node_to_node_encryption_enabled" {
   default     = false
   description = "Whether to enable node-to-node encryption"
 }
+
+locals {
```
Comment (maximmi) [inline variables.tf]: Please, move `locals` to main.tf

Comment (3h4x) [reply variables.tf:245]: I have described issue here https://github.com/cloudposse/terraform-aws-elasticsearch/issues/57

**Comment by 3h4x** [comment]
I think this PR is not required because if "*" is added to `iam_role_arns` then every role is allowed.
@janschumann Can you elaborate more on 
> but as the default aim role will be included, which results in ["<default_iam_role_arn", "*"], which causes an error.

**Comment by janschumann** [comment]
In the meantime we do not use ES anymore. Specifically the AWS service. But you might be right.

**Comment by 3h4x** [comment]
@janschumann So are you okay with closing this PR?

**Comment by osterman** [comment]
Closing for now

**Pull request closed without merging**